core/src/main/scala/cats/Foldable.scala

```diff
@@ -333,6 +333,14 @@ import Foldable.sentinel
   def foldMapM[G[_], A, B](fa: F[A])(f: A => G[B])(implicit G: Monad[G], B: Monoid[B]): G[B] =
     foldM(fa, B.empty)((b, a) => G.map(f(a))(B.combine(b, _)))
 
+  /**
+   * Equivalent to foldMapM.
+   * The difference is that foldMapA only requires G to be an Applicative
+   * rather than a Monad. It is also slower due to use of Eval.
```

Comment:
Do we actually know that this is true? My intuition is that it generally will be, but there's also some `Either` overhead involved in the `tailRecM` implementation.